[Docs] aws_elasticache_replication_group auto_minor_version_upgrade docs update (#49115)
hashicorp/terraform-provider-aws: website/docs/r/elasticache_replication_group.html.markdown

* `auth_token_update_strategy` - (Optional) Strategy used when modifying `auth_token` on an existing replication group. Not used during initial create. Valid values are `SET`, `ROTATE`, and `DELETE`. If omitted during an auth token change, AWS defaults to `ROTATE`. If value is `DELETE` then `auth_token` must be omitted.
* `auto_minor_version_upgrade` - (Optional) Specifies whether minor version engine upgrades will be applied automatically to the underlying Cache Cluster instances during the maintenance window.
  Only supported for engine types `"redis"` and `"valkey"` and if the engine version is 6 or higher.

>   Defaults to `true` (set by AWS, not the provider). If this argument is not explicitly set in the configuration, Terraform will not detect drift on this attribute.

Reviewer: I'm iffy about this suggestion, so if you disagree with it, I'd be okay with merging as is.

```suggestion
  If this argument is not explicitly set in the configuration, AWS will set a default value of `true` and Terraform will not detect drift on this attribute.
```
